lib/ui.js

```diff
@@ -2,18 +2,37 @@
 const path = require('path')
 const serve = require('serve-static')
 
-module.exports.serveRedoc = function serveRedoc (documentUrl) {
+module.exports.serveRedoc = function serveRedoc (documentUrl, opts) {
+  const toKebabCase = (string) =>
```

Comment:
Just looking around a bit I found this: https://github.com/joakimbeng/kebab-case/blob/master/index.js

Wondering if we should just use that dependency here? If we don't want that then maybe we can just hoist this outside of `serveRedoc`? I am not sure there is much of a use case for registering more than one of these middleware, but defining helper functions inside other functions like this is something I try to avoid for maintainability sake.

Reply:
I would rather keep it here so that it goes away easily when we let Redoc go. It's not going to be used anywhere else.

Reply:
I am not opposed to using a dependency, I try not to add dependencies to other people's projects though. If you are planning on moving off of redoc anyway, it is simplest to keep it here for now.

Reply:
yeah, I guess I can agree with that since we want redoc to go away.